subschemas/substrate-chain/src/resolvers/Query/crowdloan.ts

```diff
@@ -68,26 +69,34 @@ export async function activeCrowdloans(
     api.derive.chain.bestNumber(),
   ]);
 
+  const genesisHash = api.genesisHash.toHex();
   const paraIds = paraIdKeys.map(({args: [paraId]}) => paraId);
   const data = await getFunds(paraIds, bestNumber, api);
   const leasePeriod = await getLeasePeriod(api);
   const activeFunds = extractActiveFunds(data.funds, leasePeriod);
+  const startingEndpoints = createWsEndpoints((key: string, value: string | undefined) => value || key);
+  const endpoints = startingEndpoints.filter(({genesisHashRelay}) => genesisHash === genesisHashRelay);
```

Comment:
We should extract this logic to get the parachains info to a utils/service, we are doing the same in multiple places. I will create an issue for that